Website: Add meta descriptions to Fleet documentation. (#12586)
fleetdm/fleet: docs/Contributing/Configuration-for-contributors.md

  ```

<meta name="pageOrderInSection" value="1100">

> <meta name="description" value="This page contains configuration files and settings that are helpful when developing or contributing to Fleet.">

Reviewer: ```suggestion
<meta name="description" value="Learn about the configuration files and settings that are helpful when developing or contributing to Fleet.">
```
